spec/Section 7 -- Response.md

```diff
@@ -244,6 +264,77 @@ discouraged.
 }
 ```
 
+### Incremental
+
+The `incremental` entry in the response is a non-empty list of Defer or Stream
+responses. If the response of the GraphQL operation is an event stream, this
+field may appear on both the initial and subsequent values.
```

Comment:
> The `incremental` entry in the response is a non-empty list of Defer or Stream responses. 

The word response is used twice in the sentence, with completely different meanings. There is overall response, and inside it a list of responses. Hell confusing.
I suggest to adopt a different term for the second meaning, smaller response in a list - data chunk? stream entry?